src/studio/src/designer/backend/Languages/ini/en.ini

```diff
@@ -64,20 +68,24 @@ last_changed_service = Changed:
 loading_service = Loading app
 main_header = Overview
 main_subheader = Here is an overview over all the apps you have rights to edit and all apps you have rights to explore:
+make_copy = Make a copy
 new_service = new app
+new_service_copy = Name of your copy
 new_service_header = New app
 no_category_service_read = You don't have explicit read access to any apps. For a complete list, see
 no_category_service_write = You don't have write access to any apps. For a complete list, see
 no_description = The app has no description
+open_new_tab = Open new tab
+open_repository = Open repository
 search_service = Filter apps
 see_source_code = See source code
-app_already_exist = An app with that name already exists.
 service_name_description = Shown to the users and should be descriptive for what the app is about. This name can be changed later.
 service_name_has_illegal_characters = App name is illegal. Can only contain lowercase alphanumeric and hyphen (-) characters, and must begin with a letter character and end with a number or letter.
 service_name_is_too_long = App name must contain at most 30 characters
 service_owner_description = The app will be visible from the owners dashboard.
 service_saved_name_description = Used (with owner) as an unique identifier for the app. Also used as name of repository, in all URLs and in the Altinn APIs. Should therefore be both short and descriptive, such as "sick-leave" or "taxform-2019".
 sorte_services = Sort
+unknown_error_copy = An unknown error caused the copy to stop. No copy was made. If the problem persists contact <a href="mailto:tjenesteeier@altinn.no">Altinn servicedesk</a>.
```

Comment:
copy -> copying? not sure, but seems like we should use an active word here ?

Reply:
Agreed. fixed